frontend/plan/src/actions/index.js

```diff
@@ -191,7 +195,7 @@ export const loadRequirements = () => (
                         console.log(error);
                     }),
                 (error) => {
-                // eslint-disable-next-line no-console
+                    // eslint-disable-next-line no-console
                     console.log(error);
```

Comment:
I know this isn't related to this PR, but we should probably remove this log statement, right?

Reply:
In general we have pretty bad error handling practices - in the next round of refactoring we should probably figure out how to catch all these exceptions in general.